CMakeLists.txt

```diff
@@ -139,6 +145,12 @@ if(USE_CUDA)
   cuda_include_directories(BEFORE "${CMAKE_SOURCE_DIR}/third_party/thrust/dependencies/libcudacxx/include")
 endif(USE_CUDA)
 
+if(USE_ASAN)
+  message(STATUS "Use AddressSanitizer")
+  # Learn more about ASAN: https://github.com/google/sanitizers/wiki/AddressSanitizer.
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fno-omit-frame-pointer -fsanitize=leak -fsanitize-recover=all")
```

Comment:
these flags are for gcc, and not supported in MSVC?

Reply:
I think we could just append these flags to https://github.com/dmlc/dgl/blob/976970554d95d96b802faa32488d4bf836678340/CMakeLists.txt#L118

Reply:
see the above answer